### DISCUSSION TITLE
config.properties

### DISCUSSION THREAD

**Discussion by niffumau** [hassio-addons/addon-unifi#507] [General]
is there a way to create a config.properties in the site properties folder?  I need to disable WMM on the IoT network for my LIFX lights but i'm not really sure how to go about editing this file.  Normally you would ssh into the unifi server but thats not an option here.

**Comment by frenck** [comment]
> is there a way to create a config.properties in the site properties folder? 

Nope. If you need to use that, this add-on is not suitable for you.

../Frenck

**Comment by niffumau** [comment]
Oh right.  My bad.  

I found if you install the Potainer add-in for homeassistant you can then go under the container console and edit files in the container.  It gives you a bash prompt.

For anyone following along at home, to disable WMM or the 3rd wifi network I did this::
`  echo "config.system_cfg.1=wireless.3.wmm=disabled" > ./data/unifi/data/sites/default/config.properties`
Then I went into the ssid settings, enabled the hotspot portal and disabled it to force re-provisioning.  Now its disabled:
```
Wifi-4-BZ.6.6.65# fgrep wmm /tmp/running.cfg
wireless.1.wmm=enabled
wireless.2.wmm=enabled
wireless.3.wmm=enabled
wireless.4.wmm=enabled
wireless.5.wmm=enabled
wireless.3.wmm=disabled
```

Cheers, hopefully this will help some one else trying to turn of WMM

**Reply by frenck** [reply]
> I found if you install the Potainer add-in for homeassistant y

That is not supported (as a matter of fact, the use of portainer, will mark your Home Assistant instance as unsupported as well) and I will thus strongly recommend **NOT** to use that at all.

Modifying the things in an add-on manually will lead to issues as things might be in unexpected states. Additionally, if you need to go down this path, you should not be using the add-on.

I will hide your response for that reason.

../Frenck
